lms/djangoapps/certificates/models.py

```diff
@@ -854,16 +860,21 @@ class CertificateGenerationCourseSetting(TimeStampedModel):
     for the course.  This is enforced in the UI layer, but
     not in the data layer.
 
+    'language_specific_templates' controls whether certificates for the
+    course should be generated using specific language templates.
+
     """
     course_key = CourseKeyField(max_length=255, db_index=True)
-    enabled = models.BooleanField(default=False)
+    enabled = models.BooleanField(default=False)  # Deprecated
+    can_self_generate = models.BooleanField(default=False)
+    language_specific_templates = models.BooleanField(default=False)
 
     class Meta(object):
         get_latest_by = 'created'
         app_label = "certificates"
 
     @classmethod
-    def is_enabled_for_course(cls, course_key):
+    def is_self_generation_enabled_for_course(cls, course_key):
```

Comment:
Nit: I think it would be clearer without the the `is_` prefix.

Reply:
Nit: I think `course_can_self_generate` might be a little clearer, and is more consistent with the field name.